src/release/issue-triaging.md

```diff
@@ -219,14 +220,15 @@ For issues that include an Internal Compiler Error (ICE):
 - Add `A-*` labels based on the code that causes the issue (check backtraces!), and the nature of the repro (e.g. if the repro is a weird trait impl or the backtrace points to `rustc_trait_selection`, add `A-traits`).
 - Add `T-*`, `WG-*`, `PG-*`, `F-*`, `requires-*`, and `regression-*` labels as appropriate.
     - Usually the ICE should be tagged `T-compiler`. If the issue concerns the type system (e.g. trait solver), also tag the issue with `T-types`.
+- If it is an ICE in rustdoc, add `T-rustdoc` instead of `T-compiler`.
```

Comment:
Remark: there's *a bit* of nuance here, in that rustdoc is a rustc_private tool, which means that a root cause *can* be from the rustc frontend bits. But yes, I agree this is a better label in the general case, at worst the compiler label can be reapplied or adjusted.